pkg/service/group.go

```diff
@@ -102,6 +103,8 @@ func (s *groupService) createGroupRoleRelations(ctx context.Context, db bun.IDB,
 	var grs []models.GroupRole
 	var ps []*authzv1.Policy
 	var rids []uuid.UUID
+	regexc := regexp.MustCompile(`[^a-z0-9-]+`)
```

Comment:
regex does not match with the one reported in issue ( kubernetes namespace regex )

Reply:
Updated regex to `^[a-z0-9]([-a-z0-9]*[a-z0-9])?$`